exec/rpc/src/main/java/org/apache/drill/exec/rpc/RpcBus.java

```diff
@@ -164,6 +164,8 @@ public void operationComplete(ChannelFuture future) throws Exception {
       }
 
       final ChannelClosedException ex = future.cause() != null ? new ChannelClosedException(msg, future.cause()) : new ChannelClosedException(msg);
+      // close client connection to drop all temporary tables
+      clientConnection.close();
```

Comment:
Here we assume that the `close()` won't throw an exception. Clearly, it throws no checked exceptions. But, Drill likes to throw unchecked exceptions. Do we need to worry about that here? Even if such an exception were to occur; we'd still want to close the channel below.